File: templates/Uwp/Pages/CameraVB/ViewModels/CameraViewViewModel.vb
Imports Param_ItemNamespace.Helpers
Imports Param_ItemNamespace.EventHandlers

Namespace ViewModels
    Public Class CameraViewViewModel
        Inherits System.ComponentModel.INotifyPropertyChanged

        Private _photoTakenCommand As ICommand

        Private _photo As BitmapImage

        Public Property Photo As BitmapImage
            Get
                Return _photo
            End Get
            Set
                  [Param_Setter](_photo, value)
            End Set
        End Property

        Public ReadOnly Property PhotoTakenCommand As ICommand
            Get
                If _photoTakenCommand Is Nothing Then
                    _photoTakenCommand = New RelayCommand(Of CameraControlEventArgs)(AddressOf OnPhotoTaken)
                End If

                Return _photoTakenCommand
            End Get
        End Property

        Private Sub OnPhotoTaken(args As CameraControlEventArgs)
            If Not String.IsNullOrEmpty(args.Photo) Then
                Photo = New BitmapImage(New Uri(args.Photo))
            End If
        End Sub
    End Class
End Namespace

File: templates/Uwp/Pages/Camera/ViewModels/CameraViewViewModel.cs
using System;
using System.Windows.Input;
using Windows.UI.Xaml.Media.Imaging;
using Param_RootNamespace.EventHandlers;

namespace Param_RootNamespace.ViewModels
{
    public class CameraViewViewModel : System.ComponentModel.INotifyPropertyChanged
    {
        private ICommand _photoTakenCommand;
        private BitmapImage _photo;

        public BitmapImage Photo
        {
            get { return _photo; }
            set { Param_Setter(ref _photo, value); }
        }

        public ICommand PhotoTakenCommand => _photoTakenCommand ?? (_photoTakenCommand = new RelayCommand<CameraControlEventArgs>(OnPhotoTaken));

        private void OnPhotoTaken(CameraControlEventArgs args)
        {
            if (!string.IsNullOrEmpty(args.Photo))
            {
                Photo = new BitmapImage(new Uri(args.Photo));
            }
        }
    }
}
